<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>630</width>
    <height>443</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>在线翻译</string>
  </property>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>8</x>
     <y>10</y>
     <width>610</width>
     <height>424</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="2" column="0">
     <widget class="QPlainTextEdit" name="sourceLangplainTextEdit">
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>150</height>
       </size>
      </property>
     </widget>
    </item>
    <item row="1" column="1" rowspan="3">
     <widget class="QPlainTextEdit" name="targetLangplainTextEdit">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="3" column="0">
     <widget class="QPushButton" name="translateButton">
      <property name="text">
       <string>开始翻译</string>
      </property>
     </widget>
    </item>
    <item row="4" column="0" colspan="2">
     <widget class="QPlainTextEdit" name="logPlainTextEdit">
      <property name="enabled">
       <bool>false</bool>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QComboBox" name="targetLangComboBox">
      <property name="placeholderText">
       <string>选择输出的语言</string>
      </property>
     </widget>
    </item>
    <item row="0" column="0" rowspan="2">
     <widget class="QGroupBox" name="sourceLangGroupBox">
      <property name="title">
       <string>选择输入的语言</string>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QRadioButton" name="autoRadioButton">
         <property name="text">
          <string>自动检测</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="zhRadioButton">
         <property name="text">
          <string>中文</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="enRadioButton">
         <property name="text">
          <string>英文</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
